client/my-sites/upgrades/domain-management/contacts-privacy/card.jsx

```diff
@@ -57,7 +58,7 @@ const ContactsPrivacyCard = React.createClass( {
 		if ( this.props.hasPrivacyProtection && this.props.privateDomain ) {
```

Comment:
Definitely not a blocker, but it might be nice to destructure props at the start of `getNotice()`